__mocks__/adobexd/sample_output.json

```diff
@@ -0,0 +1,890 @@
+{
+  "textStyles": [
+    {
+      "token": "text-style",
+      "id": "paragraph",
+      "weight": "book",
+      "size": 18,
+      "family": "FiraSans-Book",
+      "borderBottom": null,
+      "kerning": null
+    },
+    {
+      "token": "text-style",
+      "id": "quote",
+      "weight": "bold",
+      "size": 14,
+      "family": "Helvetica-Bold",
+      "borderBottom": null,
+      "kerning": 42.9
```

Comment:
hmm can you compare to Sketch & Figma? This seems like a high value for kerning. Might be a percentage

Reply:
Yes size is incorrect if you read the values from the artboard. Should be fixed when I change this to read the textstyles from the correct file.

Reply:
The kerning in sketch is 0.6 so it's possible